ted_sws/event_manager/services/log.py

```diff
@@ -66,3 +67,9 @@ def log_mapping_suite_debug(message: str, name: str = None):
 def log_mapping_suite_warning(message: str, name: str = None):
     get_logger(name=name).warning(event_message=MappingSuiteEventMessage(message=message))
 
+
+def log_cli_brief_notice_info(message: str, name: str = None):
+    get_cli_logger(name=name).info(
+        event_message=NoticeEventMessage(message=message),
+        settings=EventMessageLogSettings(**{"briefly": True})
```

Comment:
I recommend to replace strings in code with constants.

Reply:
changed with EventMessageLogSettings(briefly=True)